src/query/functions/tests/it/scalars/testdata/hash.txt

```diff
@@ -310,11 +310,11 @@ output         : 12447382539015599056
 
 ast            : siphash64(1.1)
 raw expr       : siphash64(1.1_f64)
-checked expr   : siphash64<Float64>(1.1_f64)
-optimized expr : 14856193058643670405_u64
+checked expr   : siphash64<Decimal(38, 0)>(to_decimal<Float64>(1.1_f64))
```

Comment:
```
raw expr : siphash64(1.1_f64)
```

the 1.1_f64 should be decimal，if fixed in https://github.com/datafuselabs/databend/pull/10275